example-config.yml

```diff
@@ -386,6 +386,7 @@ itinerary:
 #   targets:
 #     testTarget: # To set a name for this target, create the i18n string "config.popups.testTarget"
 #       url: http://example.com/ 
+#       iframe: false # Whether to always launch the popup as a browser popup
```

Comment:
What is a browser popup on a phone browser?

Reply:
It just opens a new tab but it's up to the browser to pick an implementation